tests/ffi/tests.cc

```diff
@@ -605,9 +605,26 @@ extern "C" const char *cxx_run_test() noexcept {
   r_take_unique_ptr(std::unique_ptr<C>(new C{2020}));
   r_take_ref_c(C{2020});
   r_take_str(rust::Str("2020"));
+  r_take_empty_str(rust::Str(""));
+  r_take_empty_str(rust::Str(nullptr, 0));
+#ifdef CXXBRIDGE_HAS_STRING_VIEW
+  r_take_str(std::string_view("2020"));
+  r_take_empty_str(std::string_view{});
+  r_take_empty_str(std::string_view(""));
+  ASSERT(static_cast<std::string_view>(rust::Str("2020")) == "2020");
+#endif
   r_take_sliceu8(rust::Slice<const uint8_t>(
       reinterpret_cast<const uint8_t *>(SLICE_DATA), sizeof(SLICE_DATA)));
   r_take_rust_string(rust::String("2020"));
+  r_take_empty_rust_string(rust::String(""));
+  r_take_empty_rust_string(rust::String(nullptr, 0));
+#ifdef CXXBRIDGE_HAS_STRING_VIEW
+  r_take_rust_string(std::string_view("2020"));
+  r_take_empty_rust_string(std::string_view{});
+  r_take_empty_rust_string(std::string_view(""));
+  const auto rustString = rust::String("2020"); // avoid lifetime issue
```

Comment:
What lifetime issue is this comment referring to?

Reply:
"Lifetime issue" was referring to lifetime of (temporary) `rust::String("2020")` needing to exceed lifetime of `string_view` in ASSERT() below.

Was easier to use defensive programming than trying to figure it out exactly. Anyhow, checked and the temporary `rust::String("2020")` is only destructed after the comparison.

Will remove this var and merge code into ASSERT().